packages/ui/test/_mocks/components/CKEditor.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ChangeEventHandler } from 'react'
+
+import { CKEditorProps } from '@/common/components/CKEditor'
+import { Input } from '@/common/components/forms'
+
+const CKEditor = (props: CKEditorProps) => (
+  <Input
+    id={props.id || 'ck-test'}
+    name={props.id}
+    type="text"
+    autoComplete="off"
+    onChange={(props.onChange as unknown) as ChangeEventHandler<Element>}
```

Comment:
```suggestion
    onChange={(event) => onChange(event, { getData: () => event.target.value } ) }
```

I'd go with a mock that has the same interface – this way you don't need to change the production code if the tests implementation change.

I think that the test _should not_ (as long as possible) influence the production code.